packages/@dcl/sdk-commands/src/commands/deploy/index.ts

```diff
@@ -0,0 +1,208 @@
+      $ sdk-commands deploy --target my-favorite-catalyst-server.org:2323
+`
+}
+
+export async function main(options: Options) {
+  const dir = resolve(process.cwd(), options.args['--dir'] || '.')
+  const openBrowser = !options.args['--no-browser']
+  const skipBuild = options.args['--skip-build']
+  const port = options.args['--port']
+  const parsedPort = typeof port === 'string' ? parseInt(port, 10) : void 0
+  const linkerPort = parsedPort && Number.isInteger(parsedPort) ? parsedPort : void 0
+  const { error } = options.components.logger
+
+  const comps = { components: options.components }
+
+  if (!skipBuild) {
+    await npmRun(dir, 'build')
+  }
+
+  await build({ args: { '--dir': dir }, ...comps })
+
+  if (options.args['--target'] && options.args['--target-content']) {
+    throw new CliError(`You can't set both the 'target' and 'target-content' arguments.`)
+  }
+
+  // Obtain list of files to deploy
+  const ignoreFiles = (await getDCLIgnorePatterns(options.components, dir)).concat('entity.json')
+  const files = await getFiles(options.components, dir, ignoreFiles)
+  const contentFiles = new Map(files.map((file) => [file.path, file.content]))
+  const sceneJson = await validateSceneJson(options.components, dir)
+
+  const { entityId, files: entityFiles } = await DeploymentBuilder.buildEntity({
+    type: EntityType.SCENE,
+    pointers: sceneJson.scene.parcels,
+    files: contentFiles,
+    metadata: sceneJson
+  })
+
+  // Signing message
+  const messageToSign = entityId
+  const { signature, address, chainId } = await getAddressAndSignature(
+    options.components,
+    messageToSign,
+    sceneJson,
+    files,
+    {
+      openBrowser,
+      linkerPort,
+      isHttps: !!options.args['--https'],
+      skipValidations:
+        !!options.args['--skip-validations'] || !!options.args['--target'] || !!options.args['--target-content']
+    }
+  )
+  const authChain = Authenticator.createSimpleAuthChain(entityId, address, signature)
+
+  // Uploading data
+  const catalyst = await getCatalyst(options.args['--target'], options.args['--target-content'])
+
+  printProgressInfo(options.components.logger, `Uploading data to: ${catalyst.getContentUrl()}...`)
+
+  const deployData = { entityId, files: entityFiles, authChain }
+  const position = sceneJson.scene.base
+  const network = chainId === ChainId.ETHEREUM_GOERLI ? 'goerli' : 'mainnet'
+  const sceneUrl = `https://play.decentraland.org/?NETWORK=${network}&position=${position}`
+
+  try {
+    const response = (await catalyst.deploy(deployData, {
+      timeout: '10m'
+    })) as { message?: string }
+
+    if (response.message) {
+      printProgressInfo(options.components.logger, response.message)
+    }
+    printSuccess(options.components.logger, 'Content uploaded', sceneUrl)
+  } catch (e: any) {
+    error('Could not upload content:')
+    console.log(e)
+  }
+}
+
+const getCatalyst = async (target?: string, targetContent?: string) => {
```

Comment:
Let's converge towards function declarations instead of consts for functions. Also it maybe should have its own file somewhere else

```suggestion
export async function getCatalyst(target?: string, targetContent?: string) => {
```